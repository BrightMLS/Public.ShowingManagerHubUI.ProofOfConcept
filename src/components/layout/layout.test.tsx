import React from 'react'
import { shallow } from 'enzyme'
import { shallowToJson } from 'enzyme-to-json'
import Layout from './layout'

describe('<Layout /> Component', () => {
  it('should render without crashing', () => {
    const wrapper = shallow(<Layout children={null} />)
    expect(shallowToJson(wrapper)).toMatchSnapshot()
  })
})
