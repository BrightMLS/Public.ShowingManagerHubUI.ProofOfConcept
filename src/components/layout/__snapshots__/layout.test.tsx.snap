// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Layout /> Component should render without crashing 1`] = `
<WithStyles(ForwardRef(Grid))
  container={true}
>
  <WithStyles(ForwardRef(Grid))
    item={true}
    lg={2}
    md={2}
    sm={2}
    xs={12}
  >
    <SideBar />
  </WithStyles(ForwardRef(Grid))>
  <WithStyles(ForwardRef(Grid))
    className="makeStyles-root-1"
    item={true}
    lg={10}
    md={10}
    sm={10}
    xs={12}
  />
</WithStyles(ForwardRef(Grid))>
`;
